benchmark/potpie.py

```
@@ -0,0 +1,522 @@
+    }
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(url, headers=headers) as response:
+            # Check for successful response
+            if response.status == 200:
+                data = await response.json()
+                return str(data["status"])
+            else:
+                raise Exception(
+                    f"Failed to get response: {await response.text()} {response.status}"
+                )
```

Ruff (0.14.4)

Create your own exception

(TRY002)